cmd/res/docker/configuration.toml

```diff
@@ -0,0 +1,35 @@
+[Service]
```

Comment:
keeping with my suggestion this file would be located in `/cmd/edgex-ui-server/res/docker/configuration.toml`